mycroft/skills/msm_wrapper.py

```diff
@@ -59,33 +60,6 @@
 )
 
 
-def get_skills_directory(conf=None):
```

Comment:
Should this method be exposed for backwards-compat? It could just log an error and call `get_default_skills_directory`

Reply:
was added in ovos-core, not in mycroft, at the stage of development we are i think its ok to remove this one imho

Reply:
Ah, looks like it was in v0.0.1, but I never used it and idk if anyone even noticed..